src/main/java/core/basesyntax/service/operation/SupplyOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class declaration.